### PR TITLE
Fixed small visual aspects, reorganized some code, increased readability, and slightly improved loading speed.

### PR DESCRIPTION
**Pull request by TurqDevDesign**

### CODE REVIEW

**Comment by shiffman** [comment]
Thank you @TurqDevDesign! Review requested from @nilsweb then we can merge!

File: assets/css/1-tools/_fonts.sass [hunk 1]
```diff
@@ -1,7 +1,8 @@
 @import url(https://fonts.googleapis.com/css?family=Noto+Serif:400,700,400italic)
+@import url('https://fonts.googleapis.com/css?family=Open+Sans:400,400i,600,600i,700,700i')
 
 @font-face
-  font-family: 'cubanoregular'
-  src: url('../fonts/cubano-regular-webfont.woff2') format('woff2'), url('../fonts/cubano-regular-webfont.woff') format('woff')
-  font-weight: normal
-  font-style: normal
+    font-family: 'cubanoregular'
```
[the remark this answers is not in the document]
Comment (TurqDevDesign) [reply assets/css/1-tools/_fonts.sass]: My bad, I must've accidentally hit tab one too many times, thanks for catching it!

File: assets/css/2-base/_header.sass [hunk 2]
```diff
@@ -4,7 +4,7 @@ header
   .header-image
     height: 25rem
     background:
-      image: url(../images/header.jpg)
+      image: url(../images/header-progressive.jpg)
```
[the remark this answers is not in the document]
Comment (TurqDevDesign) [reply assets/css/2-base/_header.sass]: Absolutely agree, I only kept the second image just in-case we decided to go the other way, but you're right. That is the point of versioning xD

File: assets/css/4-modules/_video-card.sass [hunk 3]
```diff
@@ -2,31 +2,40 @@
   $padding: 2rem
 
   width: 35rem
-  box-shadow: 0em 0em 1em 0.5em #EEE
+  //box-shadow: 0em 0em 1em 0.5em #EEE
```
[the remark this answers is not in the document]
Comment (TurqDevDesign) [reply assets/css/4-modules/_video-card.sass]: Not a problem. I tend to comment things out for testing, then forget about them. So I'm glad you reviewed this one, haha

File: assets/javascript/functions.js [hunk 4]
```diff
@@ -57,10 +57,14 @@ window.addEventListener('load', () => {
 
   // ... attach and onClick listener to the navigation (background) to close the mobile navigation drawer.
   document.querySelector('.navigation')
-    .addEventListener('click', (event) => document.querySelector('#nav-toggle').checked = false);
+    .addEventListener('click', e => document.querySelector('#nav-toggle').checked = false);
```
[the remark this answers is not in the document]
Comment (TurqDevDesign) [reply assets/javascript/functions.js]: Actually, if I'm not mistaken, using `e` over `event` is pretty standard. If you think using `event` could improve the readability of the code, or pose some other benefit, then I'll be happy to put it back.

File: _jekyll/includes/3-modules/video-card.html [hunk 5]
```diff
@@ -17,21 +17,24 @@
...
+        {% include 1-tools/series-for-video.html video=video %}
+        <span class="video-badge">#{% if out_series.series_number%}{{ out_series.series_number }}.{% endif %}{{ include.video.video_number }}</span>
+
+        <div
+          class="video-thumbnail"
+
+          {% if include.video.video_id %}
+            style="background-image: url(https://i.ytimg.com/vi/{{include.video.video_id}}/mqdefault.jpg)"
+          {% else %}
+            style="background-image: url({{ '/assets/images/streamcover-progressive.jpg' | relative_url }})"
```
[the remark this answers is not in the document]
Comment (TurqDevDesign) [reply _jekyll/includes/3-modules/video-card.html]: Same deal as with the banner image, will do!

File: _jekyll/includes/3-modules/video-card.html [hunk 6]
```diff
@@ -17,21 +17,24 @@
...
+
+        {% comment %} The video badge (with its number). {% endcomment %}
+        {% include 1-tools/series-for-video.html video=video %}
+        <span class="video-badge">#{% if out_series.series_number%}{{ out_series.series_number }}.{% endif %}{{ include.video.video_number }}</span>
+
+        <div
+          class="video-thumbnail"
+
+          {% if include.video.video_id %}
+            style="background-image: url(https://i.ytimg.com/vi/{{include.video.video_id}}/mqdefault.jpg)"
```
[the remark this answers is not in the document]
Comment (TurqDevDesign) [reply _jekyll/includes/3-modules/video-card.html]: We'll discuss this more on Slack :)

File: assets/css/4-modules/_video-card.sass [hunk 7]
```diff
@@ -65,13 +76,15 @@
       flex-grow: 2
       margin-left: 0.25em
       margin-right: 0.25em
+      padding: 15px
+      //white-space: nowrap
```
[the remark this answers is not in the document]
Comment (TurqDevDesign) [reply assets/css/4-modules/_video-card.sass]: Absolutely! Same deal as before :P Will fix

File: assets/css/4-modules/_video-card.sass (hunk 3 above)
[the remark this answers is not in the document]
Comment (TurqDevDesign) [reply assets/css/4-modules/_video-card.sass]: Absolutely! Same deal as before :P Will fix

File: assets/javascript/functions.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (TurqDevDesign) [reply assets/javascript/functions.js]: Same as I said before, I believe `e` to be standard over `event`, but let me know what you prefer.

**Comment by nilsweb** [comment]
Just noticed that GitHub for some bizarre reason displays some comments twice... I don't know how that's happened... ¯\_(ツ)_/¯

**Comment by shiffman** [comment]
yay, so excited about this one! Thank you @TurqDevDesign and @nilsweb!

**Pull request merged**